hi-ml-azure/src/health_azure/himl.py

```diff
@@ -885,6 +869,18 @@ def submit_to_azure_if_needed(  # type: ignore
     # is necessary. If not, return to the caller for local execution.
     if submit_to_azureml is None:
         submit_to_azureml = AZUREML_FLAG in sys.argv[1:]
+
+    has_input_datasets = len(cleaned_input_datasets) > 0
+    actual_aml_workspace: Optional[Workspace] = None
```

Comment:
Why do we need `actual_` for these variables?

Reply:
Based on previous PR comments. I thought I could get the code to require fewer asserts just to satisfy mypy, but in the
end, it got as ugly as it was before. Remove the actual... now.